### ISSUE TITLE
How to know when github returns a 422

### ISSUE THREAD

**Issue by linuxgurugamer** [itchio/gothub#8]
Right now when github returns an error, the return code from gothub is 1

It would be nice if there was some way to know (other than parsing the output) what the github code was

I was hoping that the return code could be set to the github code

**Comment by mfonville** [comment]
Yes, a more detailed return code would be really useful; I now regularly get
```
error: github did not respond with 200 OK but with 403 Forbidden
```
and if the script tries to retry in that situation, the situation persists. Being able to spot that error would be really useful.
